script/pre-archive/link-assets-to-bucket.js

```diff
@@ -46,6 +46,7 @@ function linkAssetsToBucket(options, fileNames) {
 
       if (!assetSrc) continue;
       if (assetSrc.startsWith('http') || assetSrc.startsWith('data:')) continue;
+      if (assetSrc.includes('va_files')) continue;
```

Comment:
Paths containing `va_files` in their path are TeamSite assets, so they'll 404 if we try to use the bucket.